osquery/tables/system/windows/default_environment.cpp

```diff
@@ -0,0 +1,28 @@
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+static std::string kEnvironmentKey =
+    "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session "
+    "Manager\\Environment";
+
+QueryData genDefaultEnvironment(QueryContext& context) {
+  QueryData results;
+  auto environment =
+      SQL::selectAllFrom("registry", "key", EQUALS, kEnvironmentKey);
```

Comment:
Do you mind swapping this to use `queryKey` (from ./tables/system/windows/registry.h)?